class.jetpack.php

```diff
@@ -1503,23 +1507,23 @@ public static function get_active_plan() {
 		);
 
 		if ( in_array( $plan['product_slug'], $business_plans ) ) {
-			$plan['supports'] = array(
-				'videopress',
-				'akismet',
-				'vaultpress',
-				'seo-tools',
-				'google-analytics',
-				'wordads',
-				'search',
-			);
```

Comment:
As above, did you mean to remove some items from this list?

Reply:
As above - yes ;)